community/samples/serving/helloworld-java-quarkus/README.md

```diff
@@ -0,0 +1,452 @@
+---
+title: "Hello World - Java (Quarkus)"
+linkTitle: "Java"
+weight: 1
+type: "docs"
+---
+
+{{% readfile file="README.md" relative="true" markdown="true" %}}
+
+A simple [JAX-RS REST API](https://github.com/jax-rs) application that is written in Java and uses [QuarkusIO](https://quarkus.io/).
+
+It uses an ENV variable `MESSAGE_PREFIX`(defaults to value `Hello`) and prints a message of format `${MESSAGE_PREFIX} Knative World!`.
+
+## Before you begin
+
+You must meet the following requirements to run this sample:
+
+- A Kubernetes cluster with Knative installed. Follow the
+  [installation instructions](https://github.com/knative/docs/blob/master/docs/install/README.md)
+  if you need to create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured (we'll use it for a container registry).
+- You have installed
+  [Java SE 8 or later JDK](https://www.eclipse.org/openj9/).
+
+## Recreating the sample code
+
+While you can clone all of the code from this directory, hello world apps are
+generally more useful if you build them step-by-step. The following instructions
+recreate the source files from this folder.
+
+1. From the console, create a new empty web project using the maven archetype
```

Comment:
For each step, we should be clear about where you want a file created. Is this at the root? Otherwise users are forced to look at and copy the repo.

Reply:
what does it mean "is this at the root"? current verbiage seems to be inline w/ other examples, e.g., https://github.com/knative/docs/tree/master/docs/serving/samples/hello-world/helloworld-java-spring#recreating-the-sample-code